client/src/Components/index.jsx

```diff
@@ -0,0 +1,65 @@
+import Calc from "./calculator";
+import Graph from "../../../client/src/Components/graph";
+import ErrorComp from "../../../client/src/Components/errorComp";
+import { React, useState, useEffect } from "react";
+import axios from 'axios';
+
+export default function SIPCalculator() {
+  const [monthlyInvestment, setValueMonthlyInvestment] = useState(10000);
+  const [investmentPeriod, setValueInvestmentPeriod] = useState(5);
+  const [rateOfReturn, setValueRateOfReturn] = useState(10);
+  const [rateOfInflation, setValueRateOfInflation] = useState(2);
+  const [err,setError]= useState();
+  const [result, setResult] = useState();
+
+  useEffect(() => {
+    axios.get('/api', {
+      params: {
+        monthlyInvestment: monthlyInvestment,
+        investmentPeriod: investmentPeriod,
+        rateOfReturn: rateOfReturn,
+        rateOfInflation: rateOfInflation,
+      },
+    })
+    .then((res) =>{
+        if(res.data.status === -1){
```

Comment:
== serves the purpose , as typeChecking is not needed , status can be -1 as a string or number even